app/routes/_index/route.tsx

```diff
@@ -97,3 +98,49 @@ export default function HomePage() {
         </div>
     );
 }
+
+export const meta: MetaFunction = () => {
```

Comment:
Doesn't this automatically get inherited from `root`? Why does it have to be repeated here?

Reply:
Yikes. They say here that they don't do merging:
https://remix.run/docs/en/main/route/meta

So if a route returns `meta`, then only that `meta` will be used. And if it doesn't return `meta`, then parent's `meta` will be used.

Reply:
Do we need these things to defined statically for the ability to edit them through Codux?

If not, we could create a helper to reduce boilerplate.
```ts
export const meta = createMeta({
    title: '...',
    description: '...',
    image: '...'
});
```

Reply:
As far as I know right now it's not possible to edit them through Codux, but I stillI prefer having them set explicitly. so the user can easily set additional meta tags through code in a way it's described in Remix docs.